### DISCUSSION TITLE
Allow adding media from iOS Files app

### DISCUSSION THREAD

**Discussion by ghost** [community/community#47604] [Mobile]
Currently the only way to attach media to an issue on iOS is via the Photos library.

As media can be stored in the Files app too, allow adding from there as well.

Use case: I regularly take screenshots on my Mac which saves by default to the Desktop.
When iCloud backup for Desktop is turned on, screenshots are accessed via the Files app on iOS, not the Photos app.
When uploading media to GH from iOS, I can't attach media from Files, instead having to manually save the image to the Photos app from the Files app which adds unnecessary taps.

**Comment by candyho** [comment]
Hey, thanks for your feedback! I will reflect this feedback to our team and add an issue to our internal repo for tracking purpose.

**Comment by asclepiadae** [comment]
@candyho Can we encourage this as somewhat of a priority due to how long this issue has been outstanding? #10510 mentions a related issue - solving both could be done in a change!

**Reply by Orzechowski-gif** [reply]
#10510

**Reply by Orzechowski-gif** [reply]
56565656#

**Reply by iskae** [reply]
Hello @asclepiadae 👋 

Unfortunately, the https://github.com/orgs/community/discussions/10510 is related to our Web-App on mobile, but we are already supporting that feature on mobile, at least for creating an issue via sharing the image with GitHub Apps 🙇  I will also resurface your request once again and will try to prioritize it for the upcoming releases.

Please stay tuned for future releases 😊 

Cheers!
